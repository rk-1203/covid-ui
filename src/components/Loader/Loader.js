import RoundLoader from "./roundLoader.gif";

function Loader() {
  return (
    <div>
      <img src={RoundLoader} height="200px" alt="Loading..."/>
    </div>
  );
}

export default Loader;
