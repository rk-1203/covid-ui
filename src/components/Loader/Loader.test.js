import React from 'react';
import { render } from '@testing-library/react';

import Loader from "./Loader";

describe("Loader", () => {
    it("renders properly", () => {
        const { container, getByAltText } = render(<Loader />);
        expect(container.firstChild).toMatchSnapshot();
        expect(getByAltText("Loading...")).toBeInTheDocument();
    })
})