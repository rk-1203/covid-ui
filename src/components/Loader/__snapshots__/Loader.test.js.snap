// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders properly 1`] = `
<div>
  <img
    alt="Loading..."
    height="200px"
    src="roundLoader.gif"
  />
</div>
`;
